pypika/tests/test_joins.py

```diff
@@ -225,11 +300,11 @@ def test_select__star_for_table__replacement(self):
 
     def test_select_fields_with_where(self):
         q = Query.from_(self.table_abc).join(
-            self.table_efg).on(self.table_abc.foo == self.table_efg.bar
-                               ).join(
-            self.table_hij).on(self.table_abc.buz == self.table_hij.bam
-                               ).select(
-            self.table_abc.foo, self.table_efg.bar, self.table_hij.bam
+              self.table_efg).on(self.table_abc.foo == self.table_efg.bar
```

Comment:
The formatting here seems a bit odd, it's hard to see what's going on in the test. But it's actually unrelated to this pull request.

Reply:
yeah i fixed the formatting on the other tests. Didn't want to go into the whole module for this change